### PR TITLE
Proof Binary Search Variants and Catalan Orderings posts

### PR DESCRIPTION
**Pull request by kytrol**
Rinsed.

### CODE REVIEW

File: _posts/2018-08-15-Binary-Search-Useful-Variants.md [hunk 1]
```diff
@@ -41,24 +41,24 @@ and perhaps get information like the index of:
...
-instead begin searching for _earlier_ occurrences that may appear in the range `[left, mid]`. If we're interested
-in the range `[left, mid]` (inclusive) after we find one occurrence, we can just set `right = mid` to keep this
-search up. Setting `right = mid` (instead of `right = mid - 1`) includes the occurrence we just found when searching
+want to modify the logic hit when we find an occurrence to not _immediately_ return the index of `mid`. Instead, we
+should begin searching for _earlier_ occurrences that may appear in the range `[left, mid]`. If we're interested
+in the range `[left, mid]` (inclusive) after we find one occurrence, we can just set `right = mid` to continue searching.
+Setting `right = mid` (instead of `right = mid - 1`) includes the occurrence we just found when searching
 our "lower half", so in the following possible cases:
 
- - All values in the range `[left, mid - 1]` happen to be less < `arr[mid]`, we don't lose the occurrence we found
```
Comment (kytrol) [inline _posts/2018-08-15-Binary-Search-Useful-Variants.md:56]: Not sure if the < was intentional. Up to you doc.

**Review by domfarolino — APPROVED** [review]
Ty bb

File: _posts/2018-10-10-Generating-Catalan-Orderings.md [hunk 2]
```diff
@@ -89,7 +89,7 @@ As far as an implementation for this problem goes, we basically have two options
 
 We see these choices in a lot of combinatorial situations. Carrying out (1) is much easier and often requires
 much less work, since we don't actually have to enumerate over everything (+ even less so when there is a closed
-form solution). We can see it as a dynamic programming problem in which we build a table of the Catalan numbers
+form solution). We can see it as a dynamic programming, problem in which we build a table of the Catalan numbers
```
Comment (kytrol) [inline _posts/2018-10-10-Generating-Catalan-Orderings.md:92]: I'd like this comma stricken from the record.

Comment (domfarolino) [reply _posts/2018-10-10-Generating-Catalan-Orderings.md:92]: Let the defense register skepticism with the court on this ruling. "Skepticism registered". B3T

**Pull request merged**